### DISCUSSION TITLE
Reset demo data

### DISCUSSION THREAD

**Discussion by florentongit** [motor-admin/motor-admin#67] [General]
Hello,

I want to test your demo at https://app.getmotoradmin.com/demo but everything is modified, disconnected or not working. Can you reset the demo with the original data?
The version is 0.3.3 also.
Can you reset data each X hours by example?

Thank you,

**Comment by omohokcoj** [comment]
@florentongit the demo data has been reset :)

**Reply by florentongit** [reply]
@omohokcoj thank you !!!
